packages/docs/src/page-configs/ui-elements/list/examples/Clickable.vue

```diff
@@ -49,3 +50,13 @@ export default {
   },
 }
 </script>
+
+<style lang="scss">
+  #clickable-list-example {
+    padding: 0;
+
+    .list__item:not(:last-child) {
+      margin-bottom: 20px;
+    }
+  }
+</style>
```

Comment:
Let's rewrite it (in other demos too) like this:
```suggestion
<style>
.list-item + .list-item {
  margin-top: 20px;
}
</style>
```